kitipy/docker/stack.py

```diff
@@ -286,12 +287,20 @@ def get_ip_address(self, service: str, network: str):
         return data[0]['NetworkSettings']['Networks'][network]['IPAddress']
 
     def _run(self, cmd: str, **kwargs) -> subprocess.CompletedProcess:
-        kwargs.setdefault('env', self._env)
+        env = self._env.copy()
```

Comment:
Can't you encapsulate this merging vars code into a method in `BaseStack` instead of writing it 4 times?